test/e2e/framework/node/wait.go

```diff
@@ -160,32 +161,38 @@ func WaitForNodeSchedulable(ctx context.Context, c clientset.Interface, name str
 	return false
 }
 
+func WaitForNodeHeartbeatAfter(ctx context.Context, c clientset.Interface, name string, after metav1.Time, timeout time.Duration) {
```

Comment:
Can you add documentation? Please don't just repeat the function name. Paraphrase it and explain the parameters.

"nodeName" instead of "name" would be more descriptive. "after" is also very generic.

Reply:
I think `name` is enough, given the function name, package name as the context. `name` is also consistent with other functions in this package.

I've added docs to explain `after`. I think this should be enough. The type `metav1.Time` can also provide reader some context.